tests/zif/game/game_test.rb

```diff
@@ -1,27 +1,21 @@
 require 'tests/test_helpers.rb'
 
-module GameTest
-  class SceneA < Zif::Scene
+def test_unloads_scene(_args, assert)
+  test_scene_class = Class.new(Zif::Scene) do
```

Comment:
Can you describe the issue with defining a class normally here?

Reply:
Heh, I actually wrote the previous version :) I am preferring this pattern because the entire test is encapsulated and the class will go out of scope after the test is complete.